src/Symfony/Component/Lock/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+5.0.0
+-----
+
+* `Factory` has been removed, use `LockFactory` instead.
```

Comment:
there is no mention of this in 4.4

Reply:
see https://github.com/symfony/symfony/pull/32671